commons-lang/src/main/java/org/apache/commons/lang3/math/IEEE754rUtils.java

```diff
@@ -151,6 +151,8 @@ public static float min(final float a, final float b) {
     public static double max(final double... array) {
         Objects.requireNonNull(array, "array");
         Validate.isTrue(array.length != 0, "Array cannot be empty.");
+        
+        // find and return max
```

Comment:
Is this comment required?

Reply:
No, please make no other changes but just reformat the code.